Fix building with clang from meta-clang

Clang defaults to llvm runtime and this patch is needed
for building the host tools, since its using native runtime

Upstream-Status: Inappropriate [OE-Specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

--- a/src/buildtools/gn.pro
+++ b/src/buildtools/gn.pro
@@ -20,7 +20,7 @@ build_pass|!debug_and_release {
 
             gn_gen_args = --no-last-commit-position --out-path $$out_path \
                           --cc \"$$which($$CC_host)\" --cxx \"$$which($$CXX_host)\" \
-                          --ld \"$$which($$CXX_host)\" --ar \"$$which(ar)\" \
+                          --ld \"$$which($$CXX_host) -rtlib=libgcc -unwindlib=libgcc\" --ar \"$$which(ar)\" \
                           --no-static-libstdc++
 
             msvc:!clang_cl: gn_gen_args += --use-lto
